cocotb/share/lib/vpi/VpiImpl.h

```diff
@@ -127,28 +127,6 @@ class VpiReadwriteCbHdl : public VpiCbHdl {
 public:
     VpiReadwriteCbHdl(GpiImplInterface *impl);
     virtual ~VpiReadwriteCbHdl() { }
-    int run_callback(void) {
-        if (delay_kill) {
-            delay_kill = false;
-            return 0;
-        } else {
-            return VpiCbHdl::run_callback();
-        }
-    }
-    int cleanup_callback(void) {
-        if (m_state == GPI_PRIMED) {
-            delay_kill = true;
-            return 0;
-        } else {
-            return VpiCbHdl::cleanup_callback();
-        }
-    }
-    int arm_callback(void) {
-        delay_kill = false;
-        return VpiCbHdl::arm_callback();
-    }
-private:
-    bool delay_kill;
 };
```

Comment:
The changes above revert ab237e26c853f80ecc62dd0f4e9b8e70fc20b9bd, so I guess this needs re-testing on VCS

Reply:
I'm not fully sure I did the test correctly, but it should be OK (after applying #908). So let's merge this and see what happens.